# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: editor/libeditor/SplitNodeTransaction.cpp
# Commit: b9ebe58001d7
# Full Hash: b9ebe58001d787bb0bbad6d39ae8681966d5a77f
# Author: Masayuki Nakano <masayuki@d-toybox.com>
# Date: 2020-04-04 09:29:13
# Regressor Bug: 1619914
# File Overlap Count: 1
# Description:
#   Bug 1619914 - part 2: Mark transaction class methods and their caller methods as `MOZ_CAN_RUN_SCRIPT r=m_kato
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D69303
# ==============================================================================

diff -r ce2087ddab5c -r b9ebe58001d7 editor/libeditor/SplitNodeTransaction.cpp
--- a/editor/libeditor/SplitNodeTransaction.cpp	Fri Apr 03 08:30:37 2020 +0000
+++ b/editor/libeditor/SplitNodeTransaction.cpp	Fri Apr 03 08:32:01 2020 +0000
@@ -50,8 +50,7 @@
 NS_INTERFACE_MAP_BEGIN_CYCLE_COLLECTION(SplitNodeTransaction)
 NS_INTERFACE_MAP_END_INHERITING(EditTransactionBase)
 
-MOZ_CAN_RUN_SCRIPT_BOUNDARY NS_IMETHODIMP
-SplitNodeTransaction::DoTransaction() {
+NS_IMETHODIMP SplitNodeTransaction::DoTransaction() {
   if (NS_WARN_IF(!mEditorBase) || NS_WARN_IF(!mStartOfRightContent.IsSet())) {
     return NS_ERROR_NOT_AVAILABLE;
   }
@@ -118,8 +117,7 @@
   return error.StealNSResult();
 }
 
-MOZ_CAN_RUN_SCRIPT_BOUNDARY NS_IMETHODIMP
-SplitNodeTransaction::UndoTransaction() {
+NS_IMETHODIMP SplitNodeTransaction::UndoTransaction() {
   if (NS_WARN_IF(!mEditorBase) || NS_WARN_IF(!mNewLeftContent) ||
       NS_WARN_IF(!mContainerParentNode) ||
       NS_WARN_IF(!mStartOfRightContent.IsSet())) {
@@ -143,8 +141,7 @@
  * on the redo stack may depend on the left node existing in its previous
  * state.
  */
-MOZ_CAN_RUN_SCRIPT_BOUNDARY NS_IMETHODIMP
-SplitNodeTransaction::RedoTransaction() {
+NS_IMETHODIMP SplitNodeTransaction::RedoTransaction() {
   if (NS_WARN_IF(!mNewLeftContent) || NS_WARN_IF(!mContainerParentNode) ||
       NS_WARN_IF(!mStartOfRightContent.IsSet()) || NS_WARN_IF(!mEditorBase)) {
     return NS_ERROR_NOT_AVAILABLE;